src/commands/new-proj.ts

```diff
@@ -95,7 +97,82 @@ async function action() {
     if (projVm == "Docker") runExecSync(`docker build -t ${projName} ${repoPath}`);
     if (projVm == "Minikube") {
       try {
-        // Add yaml files and crate resources in another issue
+        let componentsArr: string[] = [];
```

Comment:
Split this whole action function into smaller parts. This is already 200 lines and it is still missing functionalities

Reply:
Resolving in another issue